src/xdist/remote.py

```diff
@@ -56,6 +59,45 @@ def worker_title(title):
         pass
 
 
+class RemoteMessageHandler(logging.Handler):
+    """
+    This handler sends events to a queue. Typically, it would be used together
+    with a multiprocessing Queue to centralise logging to file in one process
+    (in a multi-process application), so as to avoid file write contention
+    between processes.
+
+    This code is new in Python 3.2, but this class can be copy pasted into
+    user code for use with earlier Python versions.
+    """
+
+    def __init__(self, queue):
+        """
+        Initialise an instance, using the passed queue.
+        """
+        logging.Handler.__init__(self)
+        self.queue = queue
```

Comment:
A bit confusing calling this `queue` when it is actually a `WorkerInteractor` instance, perhaps use `worker_interactor`? Also worth updating the docs.